src/pages/AccountPage/ProfilePage.jsx

```diff
@@ -9,11 +10,12 @@ import {
   InputText,
 } from '@ynput/ayon-react-components'
 import { useUpdateUserMutation } from '../../services/user/updateUser'
-import UserDetailsHeader from '../../components/User/UserDetailsHeader'
+import Avatar from '../../components/Avatar/Avatar'
 import styled from 'styled-components'
 import UserAttribForm from '../SettingsPage/UsersSettings/UserAttribForm'
 import SetPasswordDialog from '../SettingsPage/UsersSettings/SetPasswordDialog'
 import ayonClient from '../../ayon'
+import Type from '/src/theme/typography.module.css'
```

Comment:
When the username title is back on this page, make sure to use this for the font size. `className={Type.headlineMedium}`. I know it's weird but it ensure all the correct styles are applied, like font-weight, line-height etc. It's not just font size.

Reply:
![image](https://github.com/ynput/ayon-frontend/assets/49156310/e26d86b3-1bab-494e-bfbe-434182be6930)

Reply:
Done.